tools/debug-ui/src/utils.tsx

```diff
@@ -45,8 +45,22 @@ export function formatTraffic(bytes_received: number, bytes_sent: number): React
     );
 }
 
-export function addDebugPortLink(peer_addr: string): ReactElement {
-    return <a href={'http://localhost:3000/' + peer_addr.replace(/:.*/, '/')}>{peer_addr}</a>;
+export function addDebugPortLink(peer_network_addr: string): ReactElement {
+    // Assume rpc port is always 3030 and network port is always 24567 for the first neard process on a machine
+    // Then, each subsequence neard process on the same machine will are strict increments of a fix constant from both ports
+    // peer_num should only be > 0 for nearup's localnet and 0 otherwise
+    // Reference to nearup's localnet port assumptions
+    // https://github.com/near/nearup/blob/0b9a7b60236f3164dd32677b6fa58c531a586200/nearuplib/localnet.py#L93-L94
+    // Reference to mainnet's assumption of network port
+    // https://github.com/near/nearcore/blob/e61da3d26e3614d3f6c1b669d31fca7a12d55296/chain/network/src/raw/connection.rs#L153
+    const rpc_port_assumption = 3030;
```

Comment:
I'm surprised that the code needs to be duplicated.
But if it's actually the case, then fine to leave it as is.